### DISCUSSION TITLE
Clicking on an import notification exits the play unprompted

### DISCUSSION THREAD

**Discussion by NiceAesth** [ppy/osu#19754] [Ideas]
### Type

Game behaviour

### Bug description

This should not happen, even if the game is paused. I intended to click on a screenshot notification and accidentally clicked on an import notification and was instantly exited out of the current play.

### Screenshots or videos

https://user-images.githubusercontent.com/8437201/184380432-4d58d0b6-f785-4838-8494-d894a18e9854.mp4

### Version

v2022.810.2

### Logs

[performance.log](https://github.com/ppy/osu/files/9320659/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9320660/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9320661/updater.log)
[database.log](https://github.com/ppy/osu/files/9320662/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9320663/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9320664/network.log)

**Comment by bassepeder** [comment]
This is expected behavior - it says "click to view" to be fair. Are you suggesting that this behavior should change?

**Comment by peppy** [comment]
Of note, this can't happen during actual gameplay because the notifications will not be displayed in the first place. It only happened because of intro time.

**Reply by NiceAesth** [reply]
it can still happen when pausing the game. if you take a screenshot, pause, then open notifications to view it you will be exited out of the play instantly if you accidentally misclick the import notification

**Reply by peppy** [reply]
Sure.
